doc/source/releases/20.01.rst

```diff
@@ -0,0 +1,655 @@
+  `Pull Request 8890`_
+* Refactor _set_config_base; fix config_dir value
+  `Pull Request 8915`_
+* Library tags tweaks and tests
+  `Pull Request 8947`_
+* Refactor Toolshed webapp into tool_shed package.
+  `Pull Request 8949`_
+* Separate Shed and Galaxy makos, package shed makos.
+  `Pull Request 8958`_
+* Remove admin tool shed controller code (Part 1)
+  `Pull Request 8988`_
+* Remove unused helper code from twilltestcase.
+  `Pull Request 8989`_
+* Fix tool image loading (was broken due to failing repo dir resolution)
+  `Pull Request 9006`_
+* Convert Custom builds vue
+  `Pull Request 9016`_
+* Implement datasets grid as vue component
+  `Pull Request 9018`_
+* remove unused toolshed template code
+  `Pull Request 9084`_
+* Default the authnz_manager to None
+  `Pull Request 9117`_
+* Update github username @erasche to @hexylena and correct inconsistencies
+  (thanks to `@hexylena <https://github.com/hexylena>`__).
+  `Pull Request 9194`_
+
+
+
+Fixes
+-------------------------------
+
+.. major_bug
+
+
+.. bug_tag_viz
+* Wrap SQL statement in a text() function as required by sqlalchemy
+  (thanks to `@kaukrise <https://github.com/kaukrise>`__).
+  `Pull Request 9216`_
+* Ensure visible overflow on navbar container
+  (thanks to `@kaukrise <https://github.com/kaukrise>`__).
+  `Pull Request 9217`_
+
+.. bug_tag_datatypes
+* Fix cool sniffing on python3
+  `Pull Request 9153`_
+
+.. bug_tag_tools
+* grouping: strip newlines
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 8953`_
+* Add requirements to interval_to_bed_converter.xml
+  `Pull Request 9041`_
+
+.. bug_tag_workflows
+* Exports reports with workflows (if configured).
+  `Pull Request 9055`_
+* Rev gxformat2 dependency to fix various export issues.
+  `Pull Request 9075`_
+
+.. bug_tag_ui
+* Run prettier after `#8107
+  <https://github.com/galaxyproject/galaxy/issues/8107>`__
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 9102`_
+
+.. bug_tag_jobs
+* Small fixes for k8s.
+  `Pull Request 8759`_
+* Fix output.file_path writing to read-only path on Docker
+  `Pull Request 9048`_
+* Add job home to writable paths
+  `Pull Request 9051`_
+* Fix LocalShell returncode
+  `Pull Request 9171`_
+* Fix: Added support for specifying k8s job pod priority class
+  (thanks to `@nuwang <https://github.com/nuwang>`__).
+  `Pull Request 9219`_
+* Fix: Added support for specifying k8s job affinity and tolerations
+  (thanks to `@nuwang <https://github.com/nuwang>`__).
+  `Pull Request 9225`_
+* Mount job_working_directory/configs rw
+  `Pull Request 9234`_
+* Limit async hack to async tools
+  `Pull Request 9242`_
+* Fix moving of data manager output for entries
+  `Pull Request 9243`_
+* python3 fixes for setups using drmaa and realuser setups
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 9246`_
+* fix bug introduced in https://github.com/galaxyproject/galaxy/pull/9246
```

Comment:
I guess this can be merged with the previous one. 

Btw. How can I add suggestion like @nsoranzo likes to do often, i.e. that you just need to accept a diff?

Reply:
See https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request